ports/qtbase/cmake/qt_install_submodule.cmake

```
@@ -150,7 +150,7 @@ function(qt_cmake_configure)
         ${disable_parallel}
         OPTIONS
             -DQT_NO_FORCE_SET_CMAKE_BUILD_TYPE:BOOL=ON
-            -DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON # We don't want Qt to screw with users toolchain settings.
+            -DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON # We don't want Qt to mess with users toolchain settings.

```

```suggestion
            -DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON # We don't want Qt to screw with users toolchain settings.
```
This change was cherry-picked into https://github.com/microsoft/vcpkg/pull/38682. Please revert this change, if you just modify the `scripts/cmake/vcpkg_configure_make.cmake`, you don't need modify the version file.